compose/snippets/src/main/java/com/example/compose/snippets/graphics/GraphicsModifiersSnippets.kt

```diff
@@ -485,3 +507,164 @@ fun ModifierGraphicsFlippedUsage() {
     )
     // [END android_compose_graphics_modifier_flipped_usage]
 }
+
+// [START android_compose_graphics_faded_edge_example]
+@Composable
+fun FadedEdgeBox(modifier: Modifier = Modifier, content: @Composable () -> Unit) {
+    Box(
+        modifier = modifier
+            .graphicsLayer(compositingStrategy = CompositingStrategy.Offscreen)
+            .drawWithContent {
+                drawContent()
+                drawRect(
+                    brush = Brush.verticalGradient(
+                        listOf(Color.Black, Color.Transparent)
+                    ),
+                    blendMode = BlendMode.DstIn
+                )
+            }
+    ) {
+        content()
+    }
+}
+
+@Preview
+@Composable
+private fun FadingLazyComments() {
+    FadedEdgeBox(
+        modifier = Modifier
+            .padding(32.dp)
+            .height(300.dp)
+            .fillMaxWidth()
+    ) {
+        LazyColumn {
+            items(listComments, key = { it.key }) {
+                Row(modifier = Modifier.padding(bottom = 8.dp)) {
+                    val strokeWidthPx = with(LocalDensity.current) {
+                        2.dp.toPx()
+                    }
+                    Avatar(strokeWidth = strokeWidthPx, modifier = Modifier.size(48.dp)) {
+                        Image(
+                            painter = painterResource(id = it.avatar),
+                            contentScale = ContentScale.Crop,
+                            contentDescription = null
+                        )
+                    }
+                    Spacer(Modifier.width(6.dp))
+                    Text(
+                        it.text,
+                        fontSize = 20.sp,
+                        modifier = Modifier.align(Alignment.CenterVertically)
+                    )
+                }
```

Comment:
These lines of code aren't really relevant to the API you're demonstrating, and you still can't copy-paste the whole snippet as you don't have the Avatar implementation. You could consider replacing this with:

```
items(listComments, key = { it.key }) {
  ListCommentItem(it)
}
```